index.js

```diff
@@ -110,66 +78,94 @@ function fastifyCors (fastify, opts, next) {
         )
       }
 
-      next()
+      return next()
```

Comment:
Why are we returning the next call?

Reply:
that is probably an artifact from my refactor... I can remove it :smile:

Reply:
Actually, that makes this return consistent with all of the other return paths. Does it matter? In any case, I would propose that we are consistent. I'll note the other places where next is returned...

Reply:
I don't have an opinion, I propose just that we use the same pattern everywhere. Let me know what is best...

Reply:
Good point! I reviewed this on mobile so it was tough to see how other lines were doing it.

Reply:
No worries, I’m glad to see you noticed the change to an unassuming line like this 😉